janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -163,9 +147,16 @@
             new ConfigOption<>(ES_CREATE_NS, "use-external-mappings",
             "Whether JanusGraph should make use of an external mapping when registering an index.", ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Boolean> USE_ALL_FIELD =
+        new ConfigOption<>(ELASTICSEARCH_NS, "use-all-field",
+            "Whether JanusGraph should add an \"all\" field mapping. When enabled field mappings will " +
+            "include a \"copy_to\" parameter referencing the \"all\" field. This is supported since Elasticsearch 6.x " +
+            " and is required when using wildcard fields starting in Elasticsearch 6.x.", ConfigOption.Type.GLOBAL_OFFLINE, true);
```

Comment:
@davidclement90 Now I think this should be `ConfigOption.type.FIXED` (see [here](http://docs.janusgraph.org/latest/config-ref.html#cfg-mutability)), since I don't think it should be changed once the index is created. Do you agree? Should your `USE_DEPRECATED_MULTITYPE_INDEX` be "FIXED" as well?

Reply:
No I think is GLOBAL_OFFLINE because you can stop all JanusGraph instances. Then you can upgrade ES or change directly on ES the mapping. But you may have to drop indexes for that. 
Then you change JanusGraph properties and you use reindex job if you had to drop indexes.

So I think is not FIXED because you do not change the primary storage so you can always reindex. And for the reason I think that no backend search property has to be FIXED